components/script/dom/windowproxy.rs

```diff
@@ -602,12 +602,23 @@ impl WindowProxy {
     }
 
     pub fn set_currently_active(&self, window: &Window) {
-        let globalscope = window.upcast();
+        let globalscope = window.upcast::<GlobalScope>();
+        let dest_pipeline_id = globalscope.pipeline_id();
+        if let Some(pipeline_id) = self.currently_active() {
+            if pipeline_id == dest_pipeline_id {
+                return warn!(
```

Comment:
Make this a `debug!` rather than a `warn!`?

Reply:
Ok, done.